File: Expressions/FinacialCalculator.c
//Evelyn Chennault, Financial Calculator C
#include <stdio.h>

int main(void){
    float income;
    float rent;
    float utilities;
    float groceries;
    float transportation;

    //tells user what the program does
    printf("Hello! I am a financial calculator. Let's begin! ");
    printf("I will ask you several questions. Please answer them in terms of US dollars. ");

    //questions for the user to fill out in order for the calculator to get the variables
    printf("What is your income?\n");
    scanf("%f", &income);
    printf("What is your rent?\n");
    scanf("%f", &rent);
    printf("How much do you spend on utilities?\n");
    scanf("%f", &utilities);
    printf("How much do you spend on groceries?\n");
    scanf("%f", &groceries);
    printf("How much do you spend on transportation?\n");
    scanf("%f", &transportation);

    float incomePercent = income*.1;
    float rentPercent = rent/income*100;
    float utilitiesPercent = utilities/income*100;
    float groceriesPercent = groceries/income*100;
    float transportationPercent = transportation/income*100;
    float savings = income*.1;
    float spendings = income-rent-utilities-groceries-transportation-savings;


    //all final calculations
    printf("%f is 10 percent of your income.", incomePercent);
    printf("Your rent is %f, which is %f of your income.", income, rentPercent);
    printf("Your spendings on utilities is %f, which is %f of your income.", utilities, utilitiesPercent);
    printf("Your spendings on groceries is %f, which is %f of your income.", groceries, groceriesPercent);
    printf("Your spendings on transportation is %f, which is %f of your income.", transportation, transportationPercent);
    printf("You spend %f dollars monthly.", spendings);
    printf("Your savings are 10% of your income, which is %f dollars.", savings);
    
    //ending
    printf("Thank you and have a good day! I hope we meet again.");
    return 0;
}

File: Expressions/FinacialCalculator.py
# Evelyn Chennault, Financial Calculator Python

print("Hello, user. I am a finacial calculator. I will calculate your spendings and savings. Let's begin!\n")

# user inputs
income = float(input("What is your income?\n"))
rent = float(input("What is your rent?\n"))
utilites = float(input("How much money do you pay for utilites?\n"))
groceries = float(input("How much money do you pay for groceries?\n"))
transportation = float(input("How much money do you pay for transportation?\n"))

print("\n")

savings = income*.1
spendings = (rent+utilites+groceries+transportation)/income


rent_percent = (round(rent/income))*100
utilites_percent = (utilites/income)*100
groceries_percent = (groceries/income)*100
transportation_percent = (transportation/income)*100
savings_percent = (savings/income)*100
spendings_percent = (spendings/income)*100


print(savings, "is 10 percent of your income.")
print(spendings, "which is", spendings_percent, "percent of your income.")
print("Your rent is", rent, "which is", rent_percent, "percent of your income.")
print("Your utilies are", utilites, "which is", utilites_percent, "percent of your income.")
print("Your groceries are", groceries, "which is", groceries_percent, "percent of your income.")
print("Your transportation is", transportation, "which is", transportation_percent, "percent of your income.")
print("Your savings are", savings, "which is", savings_percent, "percent of your income.")
print("Your spendings are", spendings, "which is", spendings_percent, "percent of your income.")